src/views/Dashboard.vue

```diff
@@ -293,7 +290,8 @@ export default {
 
   mounted (): void {
     // Launch Darkly flag usage example:
-    // console.log('coopsVersion =', this.flags.coopsVersion)
+    // First import common class FeatureFlags
+    // console.log('bcrs-create-ui-enabled =', featureFlags.getFlag('bcrs-create-ui-enabled'))
   },
```

Comment:
Looks like all of `mounted()` can be deleted :)

Reply:
Done.